torch/csrc/jit/serialization/mobile_bytecode.fbs

```diff
@@ -0,0 +1,197 @@
+namespace torch.jit.mobile.serialization;
+
+struct Int {
+  int_val:long;
+}
+
+struct Bool {
+  bool_val:bool;
+}
+
+struct Double{
+  double_val:double;
+}
+
+struct PerTensorAffineSchema {
+  q_scale:double;
+  q_zero_point:int;
+}
+
+table QuantizedSchema {
+  qscheme:byte;
+  scale:double;
+  zero_point:int;
+  scales:TensorMetadata;
+  zero_points:TensorMetadata;
+  axis:int;
+}
+
+table TensorMetadata {
+  // torch._utils _rebuild_tensor_v2
+  storage_location_index: uint;
```

Comment:
I don't know much about fbs schema, but looks like this file has an inconsistent with regard to whitespace after colon. In this table it is used, but say in QuantizedSchema there are no spaces between name and type of the field
```suggestion
  storage_location_index:uint;
```

Reply:
done.